cppcache/src/Log.cpp

```diff
@@ -369,6 +372,46 @@ LogLevel Log::charsToLevel(const std::string& chars) {
   }
 }
 
+void Log::setThreadName(const std::string& threadName) {
```

Comment:
Change signature to `std::string threadName` then `std::move` into `g_threadName`.

Reply:
Need to leave this alone because of the try/except block. If string is not const we get this error:

Error	C2712	Cannot use __try in functions that require object unwinding	apache-geode-static	C:\temp\geode-native\cppcache\src\Log.cpp	413